rollup.config.js

```diff
@@ -176,6 +176,7 @@ module.exports = function (args) {
     ...(c === 'index' ? [createDeclarationConfig(`src/${c}.ts`, 'dist')] : []),
     createCommonJSConfig(`src/${c}.ts`, `dist/${c}`, clientOnly),
     createESMConfig(`src/${c}.ts`, `dist/esm/${c}.mjs`, clientOnly),
+    createESMConfig(`src/${c}.ts`, `dist/webpack/${c}.js`, clientOnly),
```

Comment:
will be treated as cjs, since the nearest packagejson type is commonjs

Reply:
yeah, if webpack follows "module" convention / rule.